### PR TITLE
Init scripts (DAGGY master)

### PR DESCRIPTION
**Pull request by akomakom**
Same as #1073 but for master.
@myronkscott I picked a random location, feedback welcome
FYI @jhouserizer

### CODE REVIEW

**Comment by myronkscott** [comment]
So this allows only one tc-server per host?

File: terracotta-kit/src/assemble/init/sysv/terracotta.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+}
+
+case "$1" in
+# Start command
+start)
+  if findpid; then
+    echo "Already running"
+  else
+    echo "Starting $APP"
+    /bin/su -m $USER -c "$TERRACOTTA_HOME/bin/start-tc-server.sh &> /dev/null"
```
Comment (myronkscott) [inline terracotta-kit/src/assemble/init/sysv/terracotta.sh]: -f and -n args are often specified

File: terracotta-kit/src/assemble/init/sysv/terracotta.properties [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+USER=terracotta
+
+# Location of application's bin directory
+TERRACOTTA_HOME=/opt/terracotta
+
+# process name to grep for (in "jps -m -l" output).  Customize if running multiple instances
+JPS_MATCH="com.tc.server.TCServerMain"
+
+# Args to pass to terracotta startup script
+TC_ARGS="-n $APP -f $TERRACOTTA_HOME/config/tc.properties"
```
Comment (myronkscott) [inline terracotta-kit/src/assemble/init/sysv/terracotta.properties]: TC_ARGS="-n $NAME -f $TERRACOTTA_HOME/config/tc-config.xml"

File: terracotta-kit/src/assemble/init/sysv/terracotta.properties [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Configuration file for the Terracotta SysV init script
+# Intended to be placed into /etc/default/terracotta
+
+# Define some variables
+# Name of app
+APP=terracotta
+
+# Name of the user to run as
+USER=terracotta
+
```
Comment (myronkscott) [inline terracotta-kit/src/assemble/init/sysv/terracotta.properties]: NAME=server1

Comment (myronkscott) [reply terracotta-kit/src/assemble/init/sysv/terracotta.properties]: or NAME=server2

File: terracotta-kit/src/assemble/init/sysv/terracotta.properties (hunk 2 above)
Comment (akomakom) [reply terracotta-kit/src/assemble/init/sysv/terracotta.properties]: what's $NAME ?

File: terracotta-kit/src/assemble/init/sysv/terracotta.properties (hunk 3 above)
Comment (akomakom) [reply terracotta-kit/src/assemble/init/sysv/terracotta.properties]: is this meant to replace $APP or be a different config item?

File: terracotta-kit/src/assemble/init/sysv/terracotta.properties (hunk 2 above)
Comment (myronkscott) [reply terracotta-kit/src/assemble/init/sysv/terracotta.properties]: there are multiple servers specified in the tc-config file.  one of the servers needs to be designated for the invocation.

Comment (myronkscott) [reply terracotta-kit/src/assemble/init/sysv/terracotta.properties]: you do that with the -n switch

File: terracotta-kit/src/assemble/init/sysv/terracotta.properties (hunk 3 above)
Comment (myronkscott) [reply terracotta-kit/src/assemble/init/sysv/terracotta.properties]: this is an addition

**Comment by myronkscott** [comment]
please squash and attach to commit 5378a4b168.  Requesting daggy to master and release/5.6

**Pull request merged**